addons/hr_homeworking/models/hr_work_location.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class WorkLocation(models.Model):
+    _inherit = "hr.work.location"
+
+    location_type = fields.Selection([
+        ('home','Home'),
+        ('office','Office')], string='Cover Image', required=True)
```

Comment:
There's a lot of linting issues of that kind. Can you please install one? 
```suggestion
    location_type = fields.Selection([
        ('home', 'Home'),
        ('office', 'Office')], string='Cover Image', required=True)
```